README.rst

```diff
@@ -42,19 +63,46 @@ Add the AutoExtract downloader middleware in the settings file::
 
 Note that this should be the last downloader middleware to be executed.
 
+The providers
+-------------
 
-Usage
-=====
+Another way of consuming AutoExtract API is using the Page Objects pattern
+proposed by the `web-poet`_ library and implemented by `scrapy-poet`_.
 
-The middleware is opt-in and can be explicitly enabled per request,
-with the ``{'autoextract': {'enabled': True}}`` request meta.
-All the options below can be set either in the project settings file,
-or just for specific spiders, in the ``custom_settings`` dict.
+Page Objects their returned Items are defined by the `autoextract-poet`_
+library.
+
+Within the spider, consuming the AutoExtract result is as easy as::
+
+    from autoextract_poet.page_inputs import AutoExtractArticleData
+
+    def parse(self, response, article: AutoExtractArticleData):
+        yield article.to_item()
+
+Configuration
+^^^^^^^^^^^^^
+
+Make sure you have `scrapy-poet`_ properly configured.
```

Comment:
Currently to unuderstand how to configure scrapy-poet user would need to 

* go to https://github.com/scrapinghub/scrapy-poet
* Scroll down to the README, click on "Documentation", get to https://scrapy-poet.readthedocs.io/en/latest/
* go to "Installation guide" (it had no information)
* go to Tutorial, and read it - only there, somewhere in the middle there was a section on how to configure scrapy-poet properly (https://scrapy-poet.readthedocs.io/en/latest/intro/tutorial.html#configuring-the-project)

What do you think about

1) changing scrapy-poet docs to make configuration much more upfront,
2) pointing to this section directly here?

It was also unclear to me if "Make sure you have `scrapy-poet`_ properly configured." is done by the example code below, if if I should try find something in scrapy-poet docs. Maybe we can change the wording a bit, e.g. "First, configure scrapy-poet, as described here. Then, enable AutoExtract providers, by putting the following code to Scrapy's settinigs.py file:".

Reply:
This is a very nice observation. I've just implemented your suggestion:

- rephrased this part of our documentation
- included a link that takes the user straight to the configuration docs of scrapy-poet

Even then, I think there's room for improvement in scrapy-poet's documentation.
Since this is kinda out of the scope of this task, I've created a new issue there: https://github.com/scrapinghub/scrapy-poet/issues/36